src/components/inherited-map/components/info-panel.tsx

```diff
@@ -15,7 +15,7 @@ const InfoPanelObservable = observer(() => {
   return (
     <div className="info-panel-wrap">
       <div className="info-panel">
-        <h3 className="h3">{area.name ?? "\u00A0"}</h3>
+        <h3 className="h3">{area.name ? area.name : "\u00A0"}</h3>
```

Comment:
`.name` cannot be `undefined` according to types. So we hit `"@typescript-eslint/no-unnecessary-condition"` here. Using `? :` allows us to insert a non-breaking space if the name is an empty string or if the typings are wrong.